v2/component_analyses.yaml

```diff
@@ -0,0 +1,288 @@
+openapi: 3.0.1
+info:
+  title: Analytics Core API Specs for Version 2
+  version: "2"
+servers:
+- url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
+  description: "Fabric8 Server"
```

Comment:
Fabric8 is an old brand name. Need to replaced with 'Dependency analytics'

Reply:
Ack!